erdpy_wallet/validator_verifier.py

```diff
@@ -0,0 +1,16 @@
+from erdpy_wallet.interfaces import IVerifiable
+from erdpy_wallet.validator_keys import ValidatorPublicKey
+
+
+class ValidatorVerifier:
+    def __init__(self, public_key: ValidatorPublicKey) -> None:
+        self.public_key = public_key
+
+    @classmethod
+    def from_string(cls, buffer_hex: str) -> 'ValidatorVerifier':
```

Comment:
a bit confusing to use, as it is not clear from the name without checking the code that the string is the public key string, but I guess can stay as is.

Reply:
Thank you, noted.